tests/integration_test.go

```diff
@@ -253,54 +259,66 @@ func TestRefund_XMRTakerCancels(t *testing.T) {
 		for {
 			select {
 			case status := <-statusCh:
-				fmt.Println("> XMRMaker got status:", status)
+				t.Log("> XMRMaker got status:", status)
 				if status.IsOngoing() {
 					continue
 				}
-
-				if status != types.CompletedRefund {
-					errCh <- fmt.Errorf("swap did not refund successfully for XMRMaker: got %s", status)
+				switch status {
+				case types.CompletedRefund:
+					// Do nothing, desired outcome
+				case types.CompletedSuccess:
+					t.Log("XMRMaker completed swap before XMRTaker's cancel took affect")
+					swapSucceeded = true
+				default:
+					errCh <- fmt.Errorf("swap did not succeed or refund for XMRMaker: status=%s", status)
```

Comment:
We're in `TestRefund_XMRTakerCancels` here.  I wasn't sure how to fix this. The existing goal, as best as I can tell, was to have Alice get a refund, but on my system the swap is going to complete successfully before she cancels it.  That is what should happen the majority of the time, isn't it?

Reply:
not sure why the swap would be successful here unless the time between initiation and t0 is too small, since the xmr-taker cancels right after `types.ETHLocked`, so the contract is never set to ready and the xmr-maker shouldn't be able to claim until t0. maybe just increase the `swapTimeout`?